conu/backend/docker/container.py

```diff
@@ -224,6 +225,14 @@ def get_port_mappings(self, port=None):
             if p.split("/")[0] == str(port):
                 return port_mappings[p]
 
+    def get_image_name(self):
```

Comment:
One nit, method is available outside of class or should be private like `_get_image_name`?

Reply:
Hmm, it would be nice. but I need to call it in `Container` constructor.